Documentation/project-docs/dogfooding.md

```diff
@@ -0,0 +1,77 @@
+# How to patch dotnet CLI with latest CoreFX
+
+This document provides the steps necessary to consume a nightly build of CoreFX
+and CoreCLR.
+
+Please note that these steps aren't necessary for official builds -- these steps
+are specficic to consuming nightlies and thus unsupported builds. Also note that
+these steps are likely to change as we're simplifying this experience. Make
+sure to consult this document often.
+
+## Update framework and CLI
+
+1. Add the CoreFX feed by editing
+   `C:\Users\XXX\AppData\Roaming\NuGet\NuGet.Config` to add this line:
```

Comment:
sounds a bit dangerous to suggest editing the global NuGet.Config (if you forget to remove it, package updates from the myget feed will show up in other projects etc.).

I think creating a new NuGet.Config in the solution root would be better.